doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -709,10 +709,40 @@ For example::
 
     curl -X DELETE -H "X-Dataverse-key: $ADMIN_API_TOKEN" "$SERVER_URL/api/datasets/1234/locks?type=pidRegister"
 
-If the dataset is not locked (or if there is no lock of the specified type), the API will exit with a warning message. 
+If the dataset is not locked (or if there is no lock of the specified type), the API will exit with a warning message.
 
 (Note that all of these API calls support both the database id and persistent identifier notation for referencing the dataset)
 
+Change file labels
+~~~~~~~~~~~~~~~~~~
+To list all the file labels for the latest version use::
+
+  GET http://$SERVER/api/datasets/$id/filelabels?key=$apiKey
+
+The same endpoint, but with POST and the following JSON, can be used to replace any text in file labels::
+
+    {
+      "pattern": "*",
+      "filesToExcludeIds": [],
+      "filesToIncludeIds": [],
+      "from": "abc",
+      "to": "bcd",
+      "preview": false
+    }
+
+===================  =============  ===========
+Name                 Type           Description
+===================  =============  ===========
+pattern              string         Pattern describing which labels should be affected by the replace operation. Label matching works in two modes. When wildcard (*) is used, the pattern is matched against the whole label and the wildcard stands for any number (including zero) of any characters. When the wildcard is not present, there is a check whether the label contains the given pattern.
+filesToExcludeIds    array<number>  List of datafile ids, which should be excluded from the replace operation.
+filesToIncludeIds    array<number>  List of datafile ids, which should be included into the replace operation.
```

Comment:
I would add: `Even if datafile does not match the pattern` (maybe in parenthesis)